source/fluid/map_frame.d

```diff
@@ -24,7 +24,9 @@ import fluid.backend;
 /// is overflow.
 enum MapDropDirection {
 
-    start, center, end, automatic
+    start, center, end, automatic,
+
+    centre = center,
```

Comment:
Is this necessary? So far I've only provided names for American English, I think it's better to be consistent.

Reply:
Well, I know it's very greedy of me to want to program in my own language.

The thing is, those of us in the Commonwealth often forget that Americans spell it this way. I think it's better to just accept it whenever someone types "centre" instead of having them waste part of their life looking for what they did wrong.

It's not "necessary" but I say it's worth it.